cobra/dcode.py

```diff
@@ -114,18 +150,64 @@ def addDcodeUri(uri):
     addDcodeProxy(proxy)
 
 
-def addDcodeServer(server, port=cobra.COBRA_PORT, ssl=False):
+def addDcodeServer(server, port=cobra.COBRA_PORT, ssl=False, msgpack=True):
     scheme = "cobra"
     if ssl:
         scheme = "cobrassl"
 
     uri = "%s://%s:%d/DcodeServer" % (scheme, server, port)
+    if msgpack:
+        uri += "?msgpack=1"
+
     addDcodeUri(uri)
 
 
+# Server-side helper function
 def enableDcodeServer(daemon=None):
     server = DcodeServer()
     if daemon:
         daemon.shareObject(server, 'DcodeServer')
         return
     cobra.shareObject(server, 'DcodeServer')
+
+
+
+def main():
```

Comment:
Should this be it's own tool? Might make it a lot easier to discover if we have something like a `cobra.tools` import/directory.

Reply:
cobra.dcode seems fine to me.  if we want to make it a standalone tool, making it something like vivbin and vdbbin is fine.
but that's not what you're suggesting and i'm not really following your complete idea.  can you clarify?

Reply:
i think the majority of dcode usage should be programmatic (eg. distributed clustered code stuff), so i don't think it needs its own script (like vivbin, vdbbin, and soon vivserver).

for things which the common user may want to do more frequently, i'm a fan of having a standard script that doesn't require calling python first.
for more advanced and/or esoteric tasks, it's find to leave them as `python -m cobra.dcode`.

if you have other suggestions, let's chat about them over booze, and make a separate PR just for that.